### ISSUE TITLE
Playlist import error

### ISSUE THREAD

**Issue by maboroshin** [stephane-r/holoplay-pwa#7]
I understand that this is under development. The developer may already know.

Playlist import/export was working. But It is an error when I load a playlist that has one song since yesterday. Of course a playlist with many songs is also an error.

[holoplay-export-2023-05-17T16 41 25.563Z.txt](https://github.com/stephane-r/holoplay-pwa/files/11500748/holoplay-export-2023-05-17T16.41.25.563Z.txt) (This is ```.json```)

```
Unexpected Application Error!
i is null

XR<@https://app.holoplay.io/static/js/main.c4153499.js:2:749120
Si@https://app.holoplay.io/static/js/main.c4153499.js:2:126461
El@https://app.holoplay.io/static/js/main.c4153499.js:2:138149
Sl@https://app.holoplay.io/static/js/main.c4153499.js:2:137355
Ol@https://app.holoplay.io/static/js/main.c4153499.js:2:136957
Ss@https://app.holoplay.io/static/js/main.c4153499.js:2:186914
wc@https://app.holoplay.io/static/js/main.c4153499.js:2:172682
gc@https://app.holoplay.io/static/js/main.c4153499.js:2:172610
yc@https://app.holoplay.io/static/js/main.c4153499.js:2:172471
ic@https://app.holoplay.io/static/js/main.c4153499.js:2:169251
cc@https://app.holoplay.io/static/js/main.c4153499.js:2:169642
Wo@https://app.holoplay.io/static/js/main.c4153499.js:2:110182
4463/oc/<@https://app.holoplay.io/static/js/main.c4153499.js:2:167184
```

**Comment by stephane-r** [comment]
Hey @maboroshin 

Thank you for your report, i'll try your json soon !

Is it an export from Invidious instance ?

**Comment by maboroshin** [comment]
I exported from holoplay-pwa.

**Comment by stephane-r** [comment]
@maboroshin fixed in next release 1.5.0

**Comment by maboroshin** [comment]
This isssue is fixed.
